<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>310</width>
    <height>321</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>310</width>
    <height>321</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>310</width>
    <height>321</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ThemeSaver</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>Icons/ThemeSaverLogo2.png</normaloff>Icons/ThemeSaverLogo2.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(247, 137, 20);
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>296</width>
      <height>296</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="MainGrid">
     <item>
      <widget class="QLabel" name="ThemeSaverLogo">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>Icons/ThemeSaverLogo2.png</pixmap>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="TextAndLogo">
       <property name="bottomMargin">
        <number>5</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="TextGrid">
         <item>
          <widget class="QLabel" name="ThemeTextLabel">
           <property name="maximumSize">
            <size>
             <width>152</width>
             <height>85</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>Icons/ThemeText.png</pixmap>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="SaverTextLabel">
           <property name="maximumSize">
            <size>
             <width>135</width>
             <height>85</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>Icons/SaverText.png</pixmap>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="ButtonsGrid">
       <property name="horizontalSpacing">
        <number>3</number>
       </property>
       <property name="verticalSpacing">
        <number>10</number>
       </property>
       <item row="1" column="0">
        <widget class="QPushButton" name="LoadSlotBtn">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>131</width>
           <height>41</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>131</width>
           <height>41</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);
font: Bold 12pt &quot;Ubuntu&quot;;
color: rgb(247, 137, 20);
border-radius:10px;</string>
         </property>
         <property name="text">
          <string>  Load Slot</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>Icons/LoadSlot.png</normaloff>Icons/LoadSlot.png</iconset>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="SaveSlotBtn">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>131</width>
           <height>41</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>131</width>
           <height>41</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);
font: Bold 12pt &quot;Ubuntu&quot;;
color: rgb(247, 137, 20);
border-radius:10px;</string>
         </property>
         <property name="text">
          <string>  Save Slot</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>Icons/Save.png</normaloff>Icons/Save.png</iconset>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="ThemeShopBtn">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>131</width>
           <height>41</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>131</width>
           <height>41</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);
font: Bold 12pt &quot;Ubuntu&quot;;
color: rgb(247, 137, 20);
border-radius:10px;</string>
         </property>
         <property name="text">
          <string> Theme Shop</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>Icons/ShoppingCart.png</normaloff>Icons/ShoppingCart.png</iconset>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="ImportSlotBtn">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>131</width>
           <height>41</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>131</width>
           <height>41</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);
font: Bold 12pt &quot;Ubuntu&quot;;
color: rgb(247, 137, 20);
border-radius:10px;</string>
         </property>
         <property name="text">
          <string>  Import Slot</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>Icons/Import.png</normaloff>Icons/Import.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="BorderLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>311</width>
      <height>321</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 5px solid white;
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>BorderLabel</zorder>
   <zorder>layoutWidget</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
